walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,42 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.sql.Time;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
```

Comment:
nit: now this function has some non-trivial logic inside. I'm pretty sure that now you know exactly what is happening inside and how algorithm works. But most certainly, if you'd be reading this code in 6 month, you'd forget details. That's why it is helpful to comment and explain the code. Maybe just a few sentences at the beginning about what is going to happen, step by step.